app/src/main/java/io/mochahub/powermeter/data/WorkoutSetDao.kt

```diff
@@ -0,0 +1,27 @@
+package io.mochahub.powermeter.data
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+import androidx.room.Update
+import io.mochahub.powermeter.models.WorkoutSet
+
+@Dao
+interface WorkoutSetDao {
+    @Query("SELECT * FROM workout_sets ORDER BY createdAt ASC")
+    fun getAll(): LiveData<List<WorkoutSet>>
+
+    @Query("SELECT * FROM workout_sets where workoutUUID= :workoutID")
+    fun getWorkoutSetsByWorkout(workoutID: String): List<WorkoutSetEntity>
```

Comment:
Can return `Flow`s here too

Reply:
Removed `getAll` as it wasn't used here https://github.com/power-meter/android-client/pull/32/commits/3ea2bfdf064be26597b35d853cc086bbac60e774